### ISSUE TITLE
[BUG]  ovh_ip_firewall_rule  json: cannot unmarshal string into Go struct field IpFirewallRuleModel.destinationPort of type int64

### ISSUE THREAD

**Issue by gdumoulin** [ovh/terraform-provider-ovh#608]
### Describe the bug
When apply plan the json can't be decoded. I've this error :

ovh_ip_firewall_rule.rule-02: Creating...
╷
│ Error: Error calling Post /ip/xxx.xxx.xxx.xxx%2F32/firewall/xxx.xxx.xxx.xxx/rule
│
│   with ovh_ip_firewall_rule.rule-02,
│   on main.tf line 20, in resource "ovh_ip_firewall_rule" "rule-02":
│   20: resource "ovh_ip_firewall_rule" "rule-02" {
│
│ json: cannot unmarshal string into Go struct field IpFirewallRuleModel.destinationPort of type int64
╵
task: Failed to run task "apply": exit status 1

### Terraform Version

OpenTofu v1.6.2

### OVH Terraform Provider Version

Run `terraform init` to show the provider(s) version.

### Affected Resource(s)

Please list the resources as a list, for example:
- ovh_ip_firewall_rule

### Terraform Configuration Files

```hcl
data "ovh_ip_firewall" "ovh-vdp-genai-fw" {
  ip             = "xxx.xxx.xxx.xxx/32"
  ip_on_firewall = "xxx.xxx.xxx.xxx"
}

data "ovh_ip_firewall_rule" "rule-00" {
  ip             = data.ovh_ip_firewall.ovh-vdp-genai-fw.ip
  ip_on_firewall = data.ovh_ip_firewall.ovh-vdp-genai-fw.ip_on_firewall
  sequence       = 0
}

resource "ovh_ip_firewall_rule" "rule-01" {
  ip             = data.ovh_ip_firewall.ovh-vdp-genai-fw.ip
  ip_on_firewall = data.ovh_ip_firewall.ovh-vdp-genai-fw.ip_on_firewall
  sequence       = 1
  action         = "permit"
  protocol       = "icmp"
}

resource "ovh_ip_firewall_rule" "rule-02" {
  ip             = data.ovh_ip_firewall.ovh-vdp-genai-fw.ip
  ip_on_firewall = data.ovh_ip_firewall.ovh-vdp-genai-fw.ip_on_firewall
  sequence       = 2
  action         = "permit"
  protocol       = "tcp"
  destination_port = 22
  source = var.gdm_ip
}

```

### Expected Behavior

The rule must be creatud on firewall and terraform ressource too.

### Actual Behavior

The rule was created on the firewall but not the terraform ressource.

### Steps to Reproduce

Steps required to reproduce the issue :
1. `terraform apply`

### Additional context

The API console indicates that the destination port is a string in the return of the POST function.

**Comment by amstuta** [comment]
@gdumoulin thanks for this issue. There is indeed a mismatch between the request and response type for the property `destinationPort` that wasn't taken into account. We'll fix this shortly.

**Comment by amstuta** [comment]
@gdumoulin will be fixed in release `0.41.1` that will be done today
